### DISCUSSION TITLE
Tv boxes and stuff

### DISCUSSION THREAD

**Discussion by thenameisluk** [velvet-os/imagebuilder#269] [General]
a discussion to discuss tv boxes related things in public without having to exchange emails directly

**Comment by thenameisluk** [comment]
@hexdump0815 do you know if the tv boxes on linux side are able to receive signals from their remotes?

i couldn't find anything on it but it would be pretty useful for me

**Reply by hexdump0815** [reply]
@thenameisluk - i never played around with that aspect myself, but from what i have read many times it should be possible to make it work even with mainline in many cases - maybe this my help to get started: https://forum.armbian.com/topic/29794-how-to-install-armbian-in-h618/page/13/#comment-203591 ... besides that i'm not sure if the required infrared (ir) kernel drivers required are already included in the velvet os kernels, so that might be something to be checked as well
